subscription-service/src/main/kotlin/com/egm/stellio/subscription/web/SubscriptionHandler.kt

```diff
@@ -70,8 +70,10 @@ class SubscriptionHandler(
     @GetMapping(produces = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])
     fun getSubscriptions(
         @RequestParam(required = false, defaultValue = "1") page: Int,
-        @RequestParam(required = false, defaultValue = SUBSCRIPTION_QUERY_PAGING_LIMIT.toString()) limit: Int
+        @RequestParam(required = false, defaultValue = SUBSCRIPTION_QUERY_PAGING_LIMIT.toString()) limit: Int,
+        @RequestParam options: Optional<String>
     ): Mono<ResponseEntity<*>> {
+        val includeSysAttrs = options.map { it.contains("sysAttrs") }.orElse(false)
```

Comment:
maybe something like `options.filter { it.contains("sysAttrs") }.isPresent` makes it more meaningful?